cmd/lncli/cmd_tlvshop.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/lightningnetwork/lnd/lntypes"
+
+	"github.com/lightningnetwork/lnd/lnrpc"
+
+	"github.com/lightningnetwork/lnd/lnrpc/invoicesrpc"
+	"github.com/urfave/cli"
+)
+
+var ctxb = context.Background()
+
+var tlvshopCommand = cli.Command{
+	Name:   "tlvshop",
+	Action: actionDecorator(tlvshop),
+}
+
+func tlvshop(ctx *cli.Context) error {
```

Comment:
this commit can be removed, cool to see how this would be used tho 👍

Reply:
removed